lib/caracal/core/relationships.rb

```diff
@@ -4,42 +4,43 @@
 
 module Caracal
   module Core
-    
-    # This module encapsulates all the functionality related to registering and 
+
+    # This module encapsulates all the functionality related to registering and
     # retrieving relationships.
     #
     module Relationships
       def self.included(base)
         base.class_eval do
-          
+
           #-------------------------------------------------------------
           # Configuration
           #-------------------------------------------------------------
-          
+
           attr_reader :relationship_counter
-          
-          
+
+
           #-------------------------------------------------------------
           # Class Methods
           #-------------------------------------------------------------
-          
+
           def self.default_relationships
             [
               { target: 'fontTable.xml',  type: :font      },
               { target: 'footer1.xml',    type: :footer    },
+              { target: 'header1.xml',    type: :header    },
```

Comment:
Why `header1` rather than just `header.xml`?

Reply:
Was following the footer convention. Not sure if there is a reason to enumerate the header filename other than there could technically be multiple.

Reply:
Tempted to keep it since it is alignment with the footer file name convention.

Reply:
Sure, not a big difference, just a tad goofy :)

Reply:
🙃